app/models/manageiq/providers/autosde/inventory/persister/storage_manager.rb

```diff
@@ -23,6 +23,26 @@ def initialize_inventory_collections
       )
     end
 
+    # san_addresses
+    add_collection(physical_infra, :san_addresses) do |builder|
+      builder.add_default_values(
+        :ems_id => ->(persister) { persister.manager.id }
+      )
+      builder.add_properties(
+        :parent_inventory_collections => [:host_initiators]
+      )
+    end
+
+    # host initiators
+    add_collection(physical_infra, :host_initiators) do |builder|
+      builder.add_default_values(
+        :ems_id => ->(persister) { persister.manager.id }
+      )
+      builder.add_properties(
+        :parent_inventory_collections => [:physical_storages]
+      )
+    end
+
```

Comment:
Lets add these to the core inventory collection here: https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/inventory/persister/builder/physical_infra_manager.rb

Reply:
Actually shouldn't these all be using the `storage` persister builder not `physical_infra` ??

Reply:
@agrare they should. I guess this is leftovers we missed since at the beginning we put autosde under `physical_infra`. I changed it to `storage`

Reply:
@agrare what is the logic in placing some functionality in the core and some in the persister? why only moving the host_initiator into https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/inventory/persister/builder/storage_manager.rb ? 

if you think we should move more stuff to the core maybe I should do it in a new PR ? or is it ok to keep it inside our provider? (I think that this is functionality that currently only our provider has)

Reply:
@llivne the idea is that host_initiators are going to be a core model and should be able to be refreshed by other providers.  By having the inventory collection definition in core it allows other providers to reference them without having to redefine the same code you have here.

The issue with physical_infra vs storage is a different one and can be a separate PR

Reply:
done. see also https://github.com/ManageIQ/manageiq/pull/20858/files#diff-f623f973e49e35b8e6526b8a41057bb3707c90ed14623c932e38054ec081fb3dR21

Reply:
done. see also https://github.com/ManageIQ/manageiq/pull/20858/files#diff-f623f973e49e35b8e6526b8a41057bb3707c90ed14623c932e38054ec081fb3dR21

Reply:
:+1: